doc/api/tls.md

```diff
@@ -457,24 +478,24 @@ information.
 Returns an object representing the type, name, and size of parameter of
 an ephemeral key exchange in [Perfect Forward Secrecy][] on a client
 connection. It returns an empty object when the key exchange is not
-ephemeral. As this is only supported on a client socket, it returns `null`
-if called on a server socket. The supported types are 'DH' and 'ECDH'. The
-`name` property is only available in 'ECDH'.
-
-Example:
+ephemeral. As this is only supported on a client socket; `null` is returned
+if called on a server socket. The supported types are `'DH'` and `'ECDH'`. The
+`name` property is available only when type is 'ECDH'.
 
-    { type: 'ECDH', name: 'prime256v1', size: 256 }
+For Example: `{ type: 'ECDH', name: 'prime256v1', size: 256 }`
 
 ### tlsSocket.getPeerCertificate([ detailed ])
 
+* `detailed` {boolean} Specify `true` to request that the full certificate
+  chain with the `issuer` property be returned; `false` to return only the
+  top certificate without the `issuer` property.
+
 Returns an object representing the peer's certificate. The returned object has
-some properties corresponding to the fields of the certificate. If the
-`detailed` argument is `true` the full chain with the `issuer` property will be
-returned, if `false` only the top certificate without the `issuer` property.
+some properties corresponding to the fields of the certificate.
```

Comment:
@indutny Out of curiosity, why didn't you make `detailed === true` the default?  The extra info isn't that expensive to generate, is it?